server/routes/index.js

```diff
@@ -1,22 +1,13 @@
-import { Router } from 'express';
 import { serve, setup } from 'swagger-ui-express';
 import apiSpec from '../../swagger.json';
+import usersRoutes from './users-routes';
+import { errorResponse } from '../utils/helpers';
 
-const router = Router();
 
-
-router.use('/api-docs', serve, setup(apiSpec));
-
-router.get('/api', (req, res) => res.status(200).json({
-  status: 200,
-  message: 'Welcome to Author Haven'
-}));
-
-router.all('/*', (req, res) => {
-  res.status(404).json({
-    status: 404,
-    message: 'This is an invalid route. Please see proper documentation'
-  });
-});
+const router = (app) => {
+  app.use('/api-docs', serve, setup(apiSpec));
+  app.use('/api/users', usersRoutes);
```

Comment:
`usersRoutes` should not be pluralized. 

A single router should suffice for routing all user related endpoints to appropriate controllers.